catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -1644,6 +1685,232 @@ public void setMetacardTypes(List<MetacardType> types) {
         this.metacardTypes = types;
     }
 
+    @Override
+    public CreateResponse create(CreateRequest createRequest) throws IngestException {
```

Comment:
Added validation to the create/update/delete operations that verify Transaction is a valid operation.